### PR TITLE
Renamed Dome to MTDome.

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

**Review by r-owen — APPROVED** [review]
Looks great!

**Review by tribeiro — CHANGES_REQUESTED** [review]
FYI, this change will break code elsewhere... I was hopping we could wait until we get a clear date for the next xml release before doing this.

**Review by tribeiro — APPROVED** [review]
Lifting restriction on merging! Go ahead and merge it away! just remember to rebase to develop before merging.

**Pull request merged**
